internal/services/postgres/postgresql_flexible_server_resource_test.go

```diff
@@ -524,6 +524,48 @@ func TestAccPostgresqlFlexibleServer_invalidStorageTierScalingStorageMbStorageTi
 	})
 }
 
+func TestAccPostgresqlFlexibleServer_updateOnlyWithStorageMb(t *testing.T) {
```

Comment:
I think this is a great start to the test coverage, yet I think there are many more scenarios around this especially if we are now setting defaults when storage mb changes. I would like to see tests for storage tier set above storage tier default value, then remove the storage tier from the configuration (e.g., storage mb set to `65536` with a storage tier of `P10` in the config file) then remove the storage tier `P10` in the config. It should then revert tier from `P10` -> `P6`, etc. Does that make sense?